.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,91 @@
+name: 🐛 Bug Report
+description: Report a bug encountered while using Cluster API
+labels: kind/bug
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: What steps did you take and what happened?
+      description: |
+        Please provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner.
+        If this matter is security related, please follow the guidelines described in https://github.com/kubernetes-sigs/cluster-api/blob/main/SECURITY_CONTACTS
+      placeholder: "[A clear and concise description on how to REPRODUCE the bug.]"
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: How can we reproduce it (as minimally and precisely as possible)?
+    validations:
+      required: false
```

Comment:
nit
```suggestion
```
since the majority of not required fields doesn't have required explicitly set to false

Reply:
Dropped it, thanks.